config/routes.rb

```diff
@@ -8,33 +10,51 @@
+  add_swagger_route 'GET', '/stages', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_stages'
+  add_swagger_route 'GET', '/templates/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_template_by_id'
+  add_swagger_route 'GET', '/templates/{template_id}/workflows', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_template_workflows'
+  add_swagger_route 'GET', '/templates', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_templates'
+  add_swagger_route 'GET', '/workflows/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflow_by_id'
+  add_swagger_route 'GET', '/workflows/{workflow_id}/requests', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflow_requests'
+  add_swagger_route 'GET', '/workflows', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflows'
+  add_swagger_route 'DELETE', '/actions/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_action'
+  add_swagger_route 'DELETE', '/groups/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_group'
+  add_swagger_route 'DELETE', '/requests/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_request'
+  add_swagger_route 'DELETE', '/stages/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_stage'
+  add_swagger_route 'DELETE', '/templates/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_template'
+  add_swagger_route 'DELETE', '/workflows/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_workflow'
+  add_swagger_route 'PUT', '/actions/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_action'
+  add_swagger_route 'PUT', '/groups/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_group'
+  add_swagger_route 'PUT', '/requests/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_request'
+  add_swagger_route 'PUT', '/stages/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_stage'
+  add_swagger_route 'PUT', '/templates/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_template'
+  add_swagger_route 'PUT', '/workflows/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_workflow'
+  add_swagger_route 'GET', '/actions/{id}', controller_name: 'approvers', constraint_name: 'approversConstraint', action_name: 'fetch_action_by_id'
```

Comment:
approver (if we consider bpm having approver role) should be able to CRUD `actions`, and RU `stages`